Mathlib/Tactic/LibrarySearch.lean

```diff
@@ -92,8 +93,20 @@ def librarySearch (mvarId : MVarId) (lemmas : DiscrTree Name) (solveByElimDepth
 def lines (ls : List MessageData) :=
   MessageData.joinSep ls (MessageData.ofFormat Format.line)
 
+open Lean.Parser.Tactic
+
+-- TODO: implement the additional options for `library_search` from Lean 3,
+-- in particular including additional lemmas
+-- with `library_search [X, Y, Z]` or `library_search with attr`,
+-- or requiring that a particular hypothesis is used in the solution, with `library_search using h`.
+syntax (name := librarySearch') "librarySearch" (" (" &"config" " := " term ")")?
+  (" [" simpArg,* "]")? (" with " (colGt ident)+)? (" using " (colGt ident)+)? : tactic
+
+-- For now we only implement the basic functionality.
+-- The full syntax is recognized, but will produce a "Tactic has not been implemented" error.
+
 open Elab.Tactic Elab Tactic in
-elab tk:"librarySearch" : tactic => do
+elab_rules : tactic | `(librarySearch%$tk) => do
```

Comment:
```suggestion
elab_rules : tactic | `(tactic| librarySearch%$tk) => do
```
This has the same issue. It was elaborating to `(librarySearch % $tk)`, i.e. the mod operator applied to two identifiers, so it definitely wouldn't have worked.